uint/src/uint.rs

```diff
@@ -841,7 +890,7 @@ macro_rules! construct_uint {
 		impl_map_from!($name, i32, i64);
 		impl_map_from!($name, isize, i64);
 
-		// Converts from big endian representation of U256
+		// Converts from big endian representation of $name
```

Comment:
This is a comment. Does macro name resolve for comments?

Reply:
Good point, seems it doesn't https://github.com/rust-lang/rust/issues/37903